actions/parse-push/src/run.ts

```diff
@@ -0,0 +1,227 @@
+import * as core from '@actions/core'
+import { exec, ExecOptions } from '@actions/exec'
+import { context, getOctokit } from '@actions/github'
+import type { WebhookPayload } from '@actions/github/lib/interfaces'
+import { EventPayloads } from '@octokit/webhooks'
+import { mkdtempSync, writeFileSync } from 'fs'
+import * as path from 'path'
```

Comment:
```suggestion
import { join } from 'path'
```

I think?

Reply:
I usually make an exception for `path` because `path.join` has more meaning than `join`. It's not the same as `array.join`.